third_party/amd/lib/TritonAMDGPUToLLVM/TargetInfo.h

```diff
@@ -58,7 +58,7 @@ class TargetInfo : public mlir::triton::TargetInfoBase {
                   StringRef message, StringRef file, StringRef func,
                   int line) const override;
 
-  bool enableLinearLayout() const override { return false; }
+  bool enableLinearLayout() const override { return true; }
```

Comment:
Did you mean to revert this back to false?

Reply:
Yup; enabled to test. Will revert back to disabled before landing.